src/java/picard/analysis/CollectJumpingLibraryMetrics.java

```diff
@@ -61,26 +61,28 @@
 )
 public class CollectJumpingLibraryMetrics extends CommandLineProgram {
     static final String USAGE_SUMMARY = "Collect jumping library metrics. ";
-    static final String USAGE_DETAILS = "This tool collects high-level metrics about the " +
-            "presence of outward-facing (jumping) and inward-facing (non-jumping) read pairs within a SAM or BAM file.<br /><br />" +
-            "For a brief primer on jumping libraries, see <a href='http://gatkforums.broadinstitute.org/discussion/6326/jumping-libraries'>" +
-            "the GATK Dictionary</a>." +
-            "<br /><br />." +
-            "This program gets all data for computation from the first read in each pair in which the mapping quality (MQ) tag " +
-            "is set with the mate's mapping quality.  If the MQ tag is not set, then the program assumes that the mate's MQ is " +
-            "greater than or equal to MINIMUM_MAPPING_QUALITY (default value is 0).<br /><br /> "+
-            "All the output metrics files are structured text files."  +
-            "<br /><br />" +
-            "<h4>Usage example:</h4>" +
-            "<pre>" +
-            "java -jar picard.jar CollectJumpingLibraryMetrics \\<br />" +
-            "      I=input.bam  \\<br />" +
-            "      O=jumping_metrics.txt" +
-            "</pre>" +
-            "<hr />" +
-            "" +
-            "Please see <a href='https://broadinstitute.github.io/picard/picard-metric-definitions.html#JumpingLibraryMetrics'>" +
-            "the JumpingLibraryMetrics documentation</a> for details and explanations of the output metrics.";
+    static final String USAGE_DETAILS = "<p>This tool collects high-level metrics about the " +
+"presence of outward-facing (jumping) and inward-facing (non-jumping) read pairs within a SAM or BAM file." +
+"For a brief primer on jumping libraries, see "+
+"<a href='http://gatkforums.broadinstitute.org/discussion/6326/jumping-libraries'>the GATK Dictionary</a></p>." +
+
+"<p>This program gets all data for computation from the first read in each pair in which the mapping quality (MQ) tag " +
+"is set with the mate's mapping quality.  If the MQ tag is not set, then the program assumes that the mate's MQ is " +
+"greater than or equal to MINIMUM_MAPPING_QUALITY (default value is 0).</p> "+
+"<p>All the output metrics files are structured text files. </p>"+

```

Comment:
I assume this was in the original text, but I wonder if it's worth mentioning. Isn't that the case for all of them? This seems relatively uninformative.